src/hooks-worker-client.coffee

```diff
@@ -29,14 +29,19 @@ class HooksWorkerClient
     @emitter = new EventEmitter
 
   start: (callback) ->
+    logger.verbose('Looking up hooks handler implementation:', @language)
     @setCommandAndCheckForExecutables (executablesError) =>
```

Comment:
Yes. But I didn't want to snowball even more than I already did 😞 It would be probably worth to make a dedicated PR to fix some of the most outstanding linter issues in Dredd codebase.